src/core_plugins/elasticsearch/lib/kibana_index_mappings.js

```diff
@@ -1,16 +1,223 @@
 export const mappings = {
+  '_default_': {
+    'dynamic': 'false'
+  },
+  dashboard: {
+    properties: {
+      description: {
+        type: 'text'
+      },
+      hits: {
+        type: 'integer'
+      },
+      kibanaSavedObjectMeta: {
+        properties: {
+          searchSourceJSON: {
+            type: 'text'
+          }
+        }
+      },
+      optionsJSON: {
+        type: 'text'
+      },
+      panelsJSON: {
+        type: 'text'
+      },
+      refreshInterval: {
+        properties: {
+          display: {
+            type: 'keyword'
+          },
+          pause: {
+            type: 'boolean'
+          },
+          section: {
+            type: 'integer'
+          },
+          value: {
+            type: 'integer'
+          }
+        }
+      },
+      timeFrom: {
+        type: 'keyword'
+      },
+      timeRestore: {
+        type: 'boolean'
+      },
+      timeTo: {
+        type: 'keyword'
+      },
+      title: {
+        type: 'text'
+      },
+      uiStateJSON: {
+        type: 'text'
+      },
+      version: {
+        type: 'integer'
+      }
+    }
+  },
   config: {
+    dynamic: true,
```

Comment:
we might be able to get away with enabled: false on this, thoughts?  buildNum is used but otherwise we aren't searching